modules/objc/generator/gen_objc.py

```diff
@@ -1662,7 +1662,7 @@ def sanitize_documentation_string(doc, type):
                h_files += [os.path.join(root, filename) for filename in fnmatch.filter(filenames, '*.h')]
                hpp_files += [os.path.join(root, filename) for filename in fnmatch.filter(filenames, '*.hpp')]
             srcfiles = h_files + hpp_files
-            srcfiles = [f for f in srcfiles if not re_bad.search(f.replace('\\', '/'))]
```

Comment:
Could you add comment to the source code with the reason why you use relative path instead of absolute. Reference to the original Bazel issue will be useful too.

Reply:
I've added a comment. https://github.com/opencv/opencv/pull/26713/commits/ee7b9481461f6f23e9be61e35b1330601d114e0c
Is it okay if I don't squash it into a single commit and force push?